packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import * as KeyCode from 'keycode-js';
+import Button from 'terra-button';
+import IconInfo from 'terra-icon/lib/icon/IconInformation';
+import DemographicsBannerDisplay from '../DemographicsBannerDisplay';
+import BackgroundTile from './_BackgroundTile';
+import SelectableTile from './_SelectableTile';
+import styles from './SelectableDemographicsBanner.module.scss';
+
+const cx = classNames.bind(styles);
+
+const Icon = <IconInfo />;
+
+const propTypes = {
+  /**
+   * Application content to display in the banner.
+   */
+  applicationContent: PropTypes.node,
+  /**
+   * Age of the person.
+   */
+  age: PropTypes.string,
+  /**
+   * The persons date of birth
+   */
+  dateOfBirth: PropTypes.string,
+  /**
+   * The person's deceased date. Will display the banner as deceased if this value is provided
+   */
+  deceasedDate: PropTypes.string,
+  /**
+   * Gender of the Person
+   */
+  gender: PropTypes.string,
+  /**
+   * The persons gestational age.
+   */
+  gestationalAge: PropTypes.string,
+  /**
+   * Additional key value identifiers of a person's demographic information
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  identifiers: PropTypes.object,
+  /**
+   * @private
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   */
+  intl: intlShape.isRequired,
+  /**
+   * Full Name of the person
+   */
+  personName: PropTypes.string,
+  /**
+   * URL of Photo to display in the banner
+   */
+  photo: PropTypes.string,
+  /**
+   * The column layout in which to present the displays.
+   */
+  postMenstrualAge: PropTypes.string,
+  /**
+   * The persons preferred first name if they have one.
+   */
+  preferredFirstName: PropTypes.string,
+  /**
+   * Callback function triggered when info button is clicked.
+   */
+  infoButtonClick: PropTypes.func.isRequired,
```

Comment:
can we update the names of these click handlers to better align with `onClick`?
```suggestion
  infoButtonOnClick: PropTypes.func.isRequired,
```

Reply:
update : [here](https://github.com/cerner/terra-core/blob/d063a9d511619a4181c537d2030c86f777acf38b/packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx#L69)